crates/re_renderer/examples/2d.rs

```diff
@@ -36,16 +36,19 @@ impl framework::Example for Render2D {
             );
         }
 
-        let rerun_logo_texture = re_ctx.texture_manager_2d.create(
-            &mut re_ctx.gpu_resources.textures,
-            &Texture2DCreationDesc {
-                label: "rerun logo".into(),
-                data: image_data.into(),
-                format: wgpu::TextureFormat::Rgba8UnormSrgb,
-                width: rerun_logo.width(),
-                height: rerun_logo.height(),
-            },
-        );
+        let rerun_logo_texture = re_ctx
+            .texture_manager_2d
+            .create(
+                &mut re_ctx.gpu_resources.textures,
+                &Texture2DCreationDesc {
+                    label: "rerun logo".into(),
+                    data: image_data.into(),
+                    format: wgpu::TextureFormat::Rgba8UnormSrgb,
+                    width: rerun_logo.width(),
+                    height: rerun_logo.height(),
+                },
+            )
+            .expect("Failed to create texture for rerun logo");
```

Comment:
separate note, but I think we should stop using `unwrap` and `expect` in our examples. Using `?` is often more readable anyways

Reply:
for the samples we want to crash quickly though I'd reason. We should avoid  `unwrap` but using `expect` can help finding issues quickly. If we propagate errors upwards via `?` you loose context

Reply:
in the viewer for sure we should almost always handle errors gracefully ending either in correect workarounds or reports (or both). But to me it feels like this isn't all that applicable for samples

Reply:
`.context("Failed to create texture for rerun logo")?` isn't that much longer than `.expect("Failed to create texture for rerun logo")`

I'm not suggesting we use `thiserror`, just that we try to make our example code nice, readable, and idiomatic for our users

Reply:
that implies forwarding anyhow errors. But for the sample code how is this much different than using expect?

Reply:
Hmm am I doing something wrong or is it not possible to use `.context` on a thiserror result? I mean that then means a lot of map_err again just like in this PR

Reply:
you are doing something wrong :)

https://docs.rs/anyhow/latest/anyhow/trait.Context.html

anyways - we'll not be doing it in this PR

Reply:
ah. ofc. same problem again. It requires `Display` of the error, that's why it didn't work here. Hnghn.